sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRegistryContentClientIntegrationTests.java

```diff
@@ -268,9 +246,9 @@ public void canUploadHugeBlobInChunks(HttpClient httpClient) throws IOException
 
         client = getBlobClient("oci-artifact", httpClient);
 
-        long size = CHUNK_SIZE * 50;
+        long size = CHUNK_SIZE * 20;
```

Comment:
Curious - what motivated reducing the size?

Reply:
tests are too long, but it does not add confidence because still much lower than RAM :)  I tested it with limited heap size to validate.